mono/metadata/locales.c

```diff
@@ -294,20 +300,20 @@ construct_culture (MonoCultureInfo *this_obj, const CultureInfoEntry *ci)
 }
 
 static MonoBoolean
-construct_region (MonoRegionInfo *this_obj, const RegionInfoEntry *ri)
+construct_region (MonoRegionInfoHandle* this_obj, const RegionInfoEntry *ri)
```

Comment:
The handle is not allocated on the managed heap, so we can pass them as raw pointer, without any problem.  Passing them as value would mean that we have to keep track of every value (every copy), as we would need to update their `obj` field.